### PR TITLE
US106434 Pass token to organization-name and organization-info

### PR DESCRIPTION
**Pull request by ciss1995**

### CODE REVIEW

**Comment by svanherk** [comment]
I think we need to add the `entity-behavior` so that `token` is a property

File: components/d2l-activity-card/d2l-activity-card.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
 import {Classes, Rels} from 'd2l-hypermedia-constants';
+import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
```
Comment (edwardg8) [inline components/d2l-activity-card/d2l-activity-card.js]: very minor about the spacing

**Comment by edwardg8** [comment]
Don't we have similar PR before?

**Comment by svanherk** [comment]
> Don't we have similar PR before?

This one was for `d2l-activity-list-item`: https://github.com/BrightspaceHypermediaComponents/activities/pull/141

**Comment by edwardg8** [comment]
Oh that make sense, thanks

File: components/d2l-activity-card/d2l-activity-card.js [hunk 2]
```diff
@@ -10,12 +12,14 @@ import 'd2l-organizations/components/d2l-organization-info/d2l-organization-info
 import 'd2l-organizations/components/d2l-organization-name/d2l-organization-name.js';
 import 'd2l-typography/d2l-typography.js';
 import SirenParse from 'siren-parser';
+import 'd2l-polymer-siren-behaviors/store/entity-behavior.js';
```
Comment (svanherk) [inline components/d2l-activity-card/d2l-activity-card.js]: I think we don't need this line if we have `siren-sdk/mixin/entity-mixin.j`, or vice versa?  @m6jones?

**Pull request closed without merging**